spaceship/lib/spaceship/test_flight/client.rb

```diff
@@ -100,9 +100,20 @@ def add_group_to_build(app_id: nil, group_id: nil, build_id: nil)
 
     def testers_for_app(app_id: nil)
       assert_required_params(__method__, binding)
-      url = "providers/#{team_id}/apps/#{app_id}/testers?limit=10000"
-      response = request(:get, url)
-      handle_response(response)
+      page_size = 40 # that's enforced by the iTC servers
+      offset = nil
+      resulting_array = []
+
+      loop do
+        url = "providers/#{team_id}/apps/#{app_id}/testers?limit=#{page_size}&sort=email&order=asc"
+        url += "&offset=#{offset}" if offset
+        response = request(:get, url)
+        result = handle_response(response)
+        resulting_array += result
+        break if result.count == 0
```

Comment:
Is there a way to page through the results with a page index and page offset?
This loop worries me. If this offset doesn't actually advance the cursor (for whatever reason, e.g. email having a comma in it) we can get stuck looping.

Reply:
Not that I found one, iTunes Connect does the same thing, so I used the same appraoch